metrics/prometheus.go

```diff
@@ -533,10 +541,34 @@ func (c *PrometheusCollector) Collect(ch chan<- prometheus.Metric) {
 }
 
 const (
-	containerLabelPrefix = "container_label_"
-	containerEnvPrefix   = "container_env_"
+	// ContainerLabelPrefix is the prefix added to all container labels added
+	// to exported metrics.
+	ContainerLabelPrefix = "container_label_"
+	// ContainerEnvPrefix is the prefix added to all env variables added to
+	// exported metrics.
+	ContainerEnvPrefix = "container_env_"
 )
 
+// DefaultContainerLabels implements ContainerLabelsFunc. It exports the
+// container name, first alias, image name as well as all its env and label
+// values.
+func DefaultContainerLabels(container *info.ContainerInfo) map[string]string {
+	set := map[string]string{"id": container.Name}
+	if len(container.Aliases) > 0 {
+		set["name"] = container.Aliases[0]
+	}
+	if image := container.Spec.Image; len(image) > 0 {
+		set["image"] = image
+	}
```

Comment:
Very last part, the env labels have never worked for me so far, they're always empty (I think that's a bug somewhere in cadvisor). When I use `docker inspect <container>` I see many envs listed for kubernetes managed containers, but I've never seen any env labels on our metrics.

Shall we just remove the env part as we're already working on that part? @jimmidyson